src/TestUtils/GcloudWrapper/CloudFunction.php

```diff
@@ -263,7 +263,7 @@ private function gcloudCommand(array $args, string $channel = null)
     }
 
     /**
-     * Run the function with the php development server.
+     * Run the function with the Functions Framework router, or php development server.
```

Comment:
This comment doesn't make sense, as both options you have here use the php development server, you're just changing the path to the file. 

Also, we are changing the way this is done anyway, so reverting the comment.
```suggestion
     * Run the function with the php development server.
```